pkg/virtctl/portforward/portforward_test.go

```diff
@@ -0,0 +1,39 @@
+package portforward_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/portforward"
+)
+
+var _ = Describe("Port forward", func() {
+	DescribeTable("ParseTarget", func(arg, targetNamespace, targetName, targetKind string, expectedError string, success bool) {
+		kind, namespace, name, err := portforward.ParseTarget(arg)
+
+		Expect(namespace).To(Equal(targetNamespace))
+		Expect(name).To(Equal(targetName))
+		Expect(kind).To(Equal(targetKind))
+
+		if success {
```

Comment:
If you check for `expectedError == ""` here, you can drop the `success` variable.